frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -62,6 +62,7 @@ export default class AccordionList extends Component {
     sections: PropTypes.array.isRequired,
 
     initiallyOpenSection: PropTypes.number,
+    globalSearch: PropTypes.bool,
```

Comment:
In Admin -> Table metadata -> Semantic type input, when searching, section names will no longer be displayed. Is this expected?

See [`master`](https://github.com/metabase/metabase/assets/6830683/b8e51e46-28d0-4060-9de5-9bd48dda9d50) vs [this PR](https://github.com/metabase/metabase/assets/6830683/631eb015-2c21-4909-b5a4-79d3d9358f10)

Reply:
Good catch: that was a bug. It now works as expected:

<img width="371" alt="Screenshot 2024-04-04 at 17 03 07" src="https://github.com/metabase/metabase/assets/1250185/8834f165-4afe-4d42-b2e4-ef4ecd53be7a">


<img width="351" alt="Screenshot 2024-04-04 at 17 04 01" src="https://github.com/metabase/metabase/assets/1250185/fb66a607-f7e0-4ad7-b88c-5b02afd4fc17">